microservices/scraper-{{cookiecutter.project_slug}}/app/routes.py

```diff
@@ -0,0 +1,23 @@
+from app import app
+from app.scraper import scrape
+import json
+
+
+@app.route("/data")
+def page_data():
```

Comment:
make this actually call ```scrape``` and then have the actual function return "No data". This should keep this page more uniform between the different scrapers.